dist/styles.css

```diff
@@ -0,0 +1,5 @@
+.moxy-next-link {
+    color: unset; /* stylelint-disable-line */
```

Comment:
do you test the color of `.moxy-next-link:hover`?

Reply:
Nope.

Reply:
I suggest changing this for `inherit`, since is the same thing that `unset` and have more support.
https://caniuse.com/#search=inherit

Reply:
Agree, will do 👍